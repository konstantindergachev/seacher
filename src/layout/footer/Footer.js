import React from 'react';
import './Footer.scss';

const Footer = () => {
  return (
    <footer className="footer">
      <span className="footer__left">&copy; 2020</span>
      <span className="footer__right">Konstantin Dergachev</span>
    </footer>
  );
};

export default Footer;
