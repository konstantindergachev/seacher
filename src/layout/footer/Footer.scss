.footer {
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: space-around;
  line-height: 32px;
  font-size: 12px;
  color: #008080;
}
