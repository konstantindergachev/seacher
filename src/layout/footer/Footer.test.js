import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('to render Footer component', () => {
    expect(shallow(<Footer />).length).toEqual(1);
  });

  it('to get the snapshot of Footer component', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
