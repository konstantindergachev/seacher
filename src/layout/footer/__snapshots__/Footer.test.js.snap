// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer to get the snapshot of Footer component 1`] = `
<footer
  className="footer"
>
  <span
    className="footer__left"
  >
    Â© 2020
  </span>
  <span
    className="footer__right"
  >
    Konstantin Dergachev
  </span>
</footer>
`;
